kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -147,7 +147,8 @@
   );
   private static final ConfigDef.Importance KEY_SOURCE_IMPORTANCE = ConfigDef.Importance.MEDIUM;
   private static final String KEY_SOURCE_DOC =
-          "Determines whether the keyfile config is the path to the credentials json, or the json itself";
+          "Determines whether the keyfile config is the path to the credentials json, the json itself " +
+                  "or it is not provided as Application Default Credentials (ADC) are used.";
```

Comment:
Nit: wording

```suggestion
          "Determines whether the " + KEYFILE_CONFIG + " config is the path to the credentials json file "
              + "or the raw json of the key itself. "
              + "If set to " + GcpClientBuilder.KeySource.APPLICATION_DEFAULT.name() + ", the "
              + KEYFILE_CONFIG + " config will be ignored and the connector will use any GCP "
              + "application default credentials that it can find on the Connect worker for authentication.";
```

Reply:
Much better with this extended explanation, but I think it's more accurate to make clear that the KEYFILE_CONFIG should not be provided (we are enforcing it at validation).

```suggestion
          "Determines whether the " + KEYFILE_CONFIG + " config is the path to the credentials json file "
              + "or the raw json of the key itself. "
              + "If set to " + GcpClientBuilder.KeySource.APPLICATION_DEFAULT.name() + ", the "
              + KEYFILE_CONFIG + " should not be provided and the connector will use any GCP "
              + "application default credentials that it can find on the Connect worker for authentication.";
```

Reply:
👍 &nbsp; agreed! That wording looks great.